src/clj/game/cards-identities.clj

```diff
@@ -74,7 +74,7 @@
                                                                 (- (get-defer-damage state side :net nil) 1)
                                                                 " more net damage")))
                                                   :effect (effect (clear-wait-prompt :runner)
-                                                                  (trash target {:cause :net})
+                                                                  (trash target {:cause :net :unpreventable true})
```

Comment:
No, this part occurs after the Runner is asked about prevention. All the `damage-defer` and `get-defer-damage` stuff was put in just to make Chronos Protocol work. Although there is still the bigger problem that Chronos Protocol is still kicking in after the Runner performs prevention. :/